lib/tasks/assets_compile.rake

```diff
@@ -32,11 +32,12 @@ namespace :scihist do
   #   https://stackoverflow.com/questions/17536023/rake-assetsprecompilenodigest-in-rails-4/26049002
   task :create_non_digest_assets => :"assets:environment"  do
     %w{404.html 500.html}.each do |file|
-      digest_relative_path = Rails.application.assets_manifest.assets[file]
-      digest_path          = Rails.root + 'public/assets' + digest_relative_path
-      destination_path     = Rails.root + 'public/' + file
+      digest_relative_path = Rails.application.assets.find_asset("404.html")&.digest_path
```

Comment:
You probably mean:
```digest_relative_path = Rails.application.assets.find_asset(file)&.digest_path```

Reply:
Oh no wait, that's not a mistake -- you're just assuming both files are in the same path, so you're using one arbitrarily.

Reply:
No, you actually caught a bug! Although you are right that it probably would work anyway for the reason you said, that's not what I meant to do. 

Since you've already merged, I'll prepare another PR!

Reply:
(And thank you, this is why I appreciate actual review! :) )

Reply:
actually that was a real bug, it probably resulted in the 500.html page being a copy of the 404 instead of it's own thing! I happen to know how to force a 500 on our current app due to unrelated bug... oh weird, no I don't, that bug went away! But yeah, it should be fixed.